chanzuckerberg/idseq-web
# Surfaces pipeline version on DD Samples Table and as a warning on the heatmap

# Description

* Add a column for pipeline version on Data Discovery's Samples Table.
* Adds a warning on the heatmap when samples use different pipeline versions.

<img width="1349" alt="Screen Shot 2020-04-24 at 3 26 02 PM" src="https://user-images.githubusercontent.com/37312125/80261714-11083180-8640-11ea-91cb-900813d035ab.png">

<img width="1349" alt="Screen Shot 2020-04-24 at 3 24 56 PM" src="https://user-images.githubusercontent.com/37312125/80261721-136a8b80-8640-11ea-9325-7cb469a354df.png">

# Tests

* Load a samples table and verify that:
	* You can select pipeline version column
	* Pipeline version shown is correct
	* The header as a proper tooltip
* Create a heatmap with samples with different pipeline version for the last run and verify that:
	* It shows a notification about the different pipeline versions
	* The notification mentions the correct pipeline versions
	* The notification can be dismissed
	* The notification disappears automatically after ~12s
